terminus-ui/select/src/select.component.ts

```diff
@@ -125,7 +134,7 @@ const DEFAULT_DELIMITER = ',';
 /**
  * Expose the formatter function type
  */
-export type TsSelectFormatFn = (v: any) => string;
+export type TsSelectFormatFn = (v: unknown) => string;
```

Comment:
It's new(ish). I think this is the first time it's made it into the library.

> Anything is assignable to unknown, but unknown isn't assignable to anything but itself and any without a type assertion or a control flow based narrowing. Likewise, no operations are permitted on an unknown without first asserting or narrowing to a more specific type.

```typescript
myFunction(v: any) {
  // Since `v` is any, TypeScript offers no protection if we don't pass
  // in the correct type
  return v.foo.bar;
}

myFunction(v: unknown) {
  // Since `v` is unknown, TypeScript will show an error here and force us
  // to actually do type checking inside the function before we can
  // reference `foo.bar`
  return v.foo.bar;
}
```